### PR TITLE
feat(console): Added component with iperf command

### PR DESCRIPTION
**Pull request by espressif-abhikroy**
The component provides a console where the 'iperf' command can be executed for any example project.

### CODE REVIEW

File: .github/workflows/console_cmd_iperf__build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+
+jobs:
+  build_console_cmd_iperf:
+    if: contains(github.event.pull_request.labels.*.name, 'console') || github.event_name == 'push'
+    name: Build
+    strategy:
+      matrix:
+        idf_ver: ["latest", "release-v5.0"]
```
Comment (euripedesrocha) [inline .github/workflows/console_cmd_iperf__build.yml:16]: Should we add 5.1?

File: components/console_cmd_iperf/.cz.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+commitizen:
+  bump_message: 'bump(console): $current_version -> $new_version'
+  pre_bump_hooks: python ../../ci/changelog.py console_cmd_iperf
+  tag_format: console_cmd_iperf-v$version
+  version: 0.0.9
```
Comment (euripedesrocha) [inline components/console_cmd_iperf/.cz.yaml:6]:
```suggestion
  version: 0.1.0
```

File: components/console_cmd_iperf/console_iperf.c [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    struct arg_int *time;
+    struct arg_int *bw_limit;
+    struct arg_lit *abort;
+    struct arg_end *end;
+} iperf_args;
+
+static int do_cmd_iperf(int argc, char **argv)
+{
+    int nerrors = arg_parse(argc, argv, (void **)&iperf_args);
+    iperf_cfg_t cfg;
```
Comment (euripedesrocha) [inline components/console_cmd_iperf/console_iperf.c:61]:
```suggestion
    /* ethernet iperf only support IPV4 address */
    iperf_cfg_t cfg = {.type = IPERF_IP_TYPE_IPV4};
```

File: components/console_cmd_iperf/console_iperf.c [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+    if (nerrors != 0) {
+        arg_print_errors(stderr, iperf_args.end, argv[0]);
+        return 0;
+    }
+
+    memset(&cfg, 0, sizeof(cfg));
```
Comment (euripedesrocha) [inline components/console_cmd_iperf/console_iperf.c:68]:
```suggestion
```

File: components/console_cmd_iperf/console_iperf.c [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+    /* ethernet iperf only support IPV4 address */
+    cfg.type = IPERF_IP_TYPE_IPV4;
```
Comment (euripedesrocha) [inline components/console_cmd_iperf/console_iperf.c:71]:
```suggestion
```

File: components/console_cmd_iperf/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+3. Ensure esp-netif and NVS flash is initialized and default event loop is created in your app_main():
+    ```c
+    ESP_ERROR_CHECK(esp_netif_init());
+    ESP_ERROR_CHECK(esp_event_loop_create_default());
+    esp_err_t ret = nvs_flash_init();   //Initialize NVS
+    if (ret == ESP_ERR_NVS_NO_FREE_PAGES || ret == ESP_ERR_NVS_NEW_VERSION_FOUND) {
+        ESP_ERROR_CHECK(nvs_flash_erase());
+        ret = nvs_flash_init();
+    }
+    ESP_ERROR_CHECK(ret);
```
Comment (david-cermak) [inline components/console_cmd_iperf/README.md:21]: I think NVS is only needed for wifi

**Review by david-cermak — APPROVED** [review]
LGTM!

**Pull request merged**